client/my-sites/themes/themes-selection.jsx

```diff
@@ -21,13 +21,17 @@ import {
 	isThemeActive,
 	isThemePurchased
 } from 'state/themes/selectors';
+import config from 'config';
 import { FEATURE_UNLIMITED_PREMIUM_THEMES } from 'lib/plans/constants';
 import { PAGINATION_QUERY_KEYS } from 'lib/query-manager/paginated/constants';
 
 const ThemesSelection = React.createClass( {
 	propTypes: {
 		query: PropTypes.object.isRequired,
-		siteId: PropTypes.number,
+		siteId: PropTypes.oneOfType( [
+			PropTypes.number,
+			PropTypes.oneOf( [ false ] )
```

Comment:
I just realized that my advise of using `false` makes for a really ugly `propType` 😄 Let's revert that `propType` change here and try `null` as `siteId` prop in `single-site-jetpack` instead (since there's no `isRequired` here, that will hopefully work without needing to modify the `propType`).